clients/py/v3io_frames/grpc.py

```diff
@@ -38,6 +40,8 @@
     ('grpc.max_receive_message_length', GRPC_MESSAGE_SIZE),
 ]
 
+warnings.formatwarning = lambda msg, *args, **kwargs: f'{msg}\n'
```

Comment:
why are you purposely ignoring args and kwargs?

Reply:
warnings prints the msg in a very ugly format - 
`/Users/dinaleventol/PycharmProjects/framestest/v3io_frames_local/http.py:254: UserWarning: Warning - Server version 'unknown' is different from client version '0.4.1'. Some operations may not work.
  __version__ + "\'. Some operations may not work.")` 

googled it a bit and found this way of stripping the extra info warnings prints and keeping just the warn msg itself